static/app/views/replays/detail/console/consoleLogRow.tsx

```diff
@@ -144,18 +127,35 @@ const ConsoleLog = styled('div')<{
 `;
 
 const ICONS = {
-  [BreadcrumbLevelType.ERROR]: <IconFire size="xs" />,
-  [BreadcrumbLevelType.WARNING]: <IconWarning size="xs" />,
+  [BreadcrumbLevelType.ERROR]: (
```

Comment:
Why did we change the error icon from "fire" to "close"?

Reply:
+1 curious about this too